scripts/setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='singularity-logfetch',
-    version='0.0.9',
+    version='0.0.10',
```

Comment:
best to set `VERSION="X.Y.Z"` in `logfetch/__init__.py` and import it here, so that we don't have 2 values floating around